services/visual-studio-app-center/visual-studio-app-center-releases-size.service.js

```diff
@@ -0,0 +1,62 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const prettyBytes = require('pretty-bytes')
+const { nonNegativeInteger } = require('../validators')
+const {
+  BaseVisualStudioAppCenterService,
+  keywords,
+  documentation,
+} = require('./visual-studio-app-center-base')
+
+const schema = Joi.object({
+  size: nonNegativeInteger.required(),
```

Comment:
No need to include the `.required()` here as that's already attached:
https://github.com/badges/shields/blob/11836f55de1541a7c55d077c6f359c20b82e7570/services/validators.js#L15

```suggestion
  size: nonNegativeInteger,
```